src/App.jsx

```diff
@@ -14,33 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SortBy = {
+  sortByAlphabet: 'alphabet',
+  sortByLength: 'length',
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const getPrepareGoods = (goods, { sortField, reversed }) => {
+  const prepareGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    switch (sortField) {
+      case SortBy.sortByAlphabet:
+        prepareGoods.sort((good1, good2) => good1.localeCompare(good2));
+        break;
+      case SortBy.sortByLength:
+        prepareGoods.sort((good1, good2) => good1.length - good2.length);
+        break;
+      default:
+        return 0;
+    }
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (reversed) {
+    prepareGoods.reverse();
+  }
+
+  return prepareGoods;
+};
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
It's better name for boolean variable - see article: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig